docs/configuration_file.md

```diff
@@ -270,6 +311,37 @@ Where:
 
 Pair converters are optional: if they are missing from the configuration file DaLI selects a default one.
 
+### CCXT
+This plugin is based on the CCXT Python library.
+
+Initialize this plugin section as follows:
+<pre>
+[dali.plugin.pair_converter.ccxt</em>]
+historical_price_type = <em>&lt;historical_price_type&gt;</em>
+fiat_priority = <em>&lt;fiat_priority&gt;</em>
+</pre>
+
+Where:
+* `<historical_price_type>` is one of `open`, `high`, `low`, `close`, `nearest`. When DaLi downloads historical market data, it captures a `bar` of data surrounding the timestamp of the transaction. Each bar has a starting timestamp, an ending timestamp, and OHLC prices. You can choose which price to select for price lookups. The open, high, low, and close prices are self-explanatory. The `nearest` price is either the open price or the close price of the bar depending on whether the transaction time is nearer the bar starting time or the bar ending time.
+* `fiat_priority` is an optional list of strings in JSON format that ranks the priority of fiat in the routing system. If no `fiat_priority` is given, the default priority is `["USD", "JPY", "KRW", "EUR", "GBP", "AUD"]`, which is based on the volume of the fiat market paired with BTC (ie. BTC/USD has the highest worldwide volume, than BTC/JPY, etc.).
+
+The CCXT pair converter plugin uses a routing system to find the shortest pricing path between a base asset and a quote asset (what the asset is priced in). It does this by assembling a graph of nodes made out of assets and edges made from markets with a preference for the exchange the asset was purchased on. Fiat exchange rates from the European Central Bank are also added to the graph to allow any fiat to be converted between each other. 
+
+For example, if a user needs the price of their BETH (Beaconed ETH) purchase on Binance.com in CHF (Swiss Francs). The router will first route the price through the available markets on the exchange:
+
+1. BETH -> ETH
+2. ETH -> GBP
+
+Then, it will route the price through a fiat conversion to get the final price:
+
+3. GBP -> CHF
```

Comment:
The example is very helpful. Maybe describe what fiat_priority is being used (the default one? ) and add a brief comment in each line describing what is happening:
e.g. in the case of ETH -> GBP, why is it picking GBP (which seems quite low in the default list)?